ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.test.tsx

```diff
@@ -77,6 +77,18 @@ const TEST_TIME_SERIES_PANEL: TimeSeriesChartProps = {
   },
 };
 
+function getLegendByName(name?: string) {
+  if (typeof name !== 'string') {
+    throw new Error('Legend name must be a string.');
+  }
+
+  return screen.getByRole('listitem', {
+    name: (content, element) => {
+      return element.innerHTML.includes(name);
+    },
```

Comment:
This is a little ugly, but I couldn't get a basic name lookup to work. It might be because of the nested dom elements, the name often including a lot of special characters, or some combination thereof. Hopefully wrapping it in a helper function with a nice name minimizes the pain.